python/ray/tune/search/sample.py

```diff
@@ -601,20 +599,19 @@ def quniform(lower: float, upper: float, q: float):
 
 
 @PublicAPI
-def loguniform(lower: float, upper: float, base: float = 10):
```

Comment:
Keep the argument but add a deprecation warning to remove it. Don't want to break people's code too quickly with a `PublicAPI`.

```python
from ray.util.annotations import RayDeprecationWarning

warnings.warn(
    (
        "The `loguniform(base)` argument is deprecated. "
        "Please remove it as it is not actually needed in this method."
    ),
    RayDeprecationWarning,
    stacklevel=2,
)
```